src/audio/PlaylistTrack.js

```diff
@@ -449,7 +511,7 @@ Object.defineProperty(FORGE.PlaylistTrack.prototype, "loop",
     /** @this {FORGE.PlaylistTrack} */
     set: function(value)
     {
-        this._loop = Boolean(value);
+        this._loop = !!value;
```

Comment:
Why change this? the Boolean(var) syntax is the one used on the project from the start.